lib/addons/nginx/index.ts

```diff
@@ -97,13 +109,20 @@ export class NginxAddOn implements ClusterAddOn {
             }
         };
 
-        clusterInfo.cluster.addHelmChart("nginx-addon", {
+        const nginxHelmChart = clusterInfo.cluster.addHelmChart("nginx-addon", {
             chart: "nginx-ingress",
             repository: "https://helm.nginx.com/stable",
             release: Constants.SSP_ADDON,
             namespace: props.namespace,
             version: props.version,
             values
         });
+
+        const asyncTasks = Promise.all(dependencies.values()).then((constructs) => {
```

Comment:
You don't need the const here, just `Promise.all(dependencies.values()).then(...).catch(err)`.

Reply:
Fixed.